chronicle-front/src/app/components/ticket-view/ticket-view.component.ts

```diff
@@ -0,0 +1,155 @@
+import { Component, OnInit } from '@angular/core';
+import { Observable } from 'rxjs';
+import { Ticket } from 'src/app/models/Ticket';
+import { Video } from 'src/app/models/Video';
+import { AuthService } from 'src/app/services/auth.service';
+import { MediaRetrievalService } from 'src/app/services/media-retrieval.service';
+import { TicketService } from 'src/app/services/ticket.service';
+
+@Component({
+  selector: 'app-ticket-view',
+  templateUrl: './ticket-view.component.html',
+  styleUrls: ['./ticket-view.component.css']
+})
+export class TicketViewComponent implements OnInit {
+
+  constructor(private ticketService: TicketService, private authService:AuthService) { }
+  ngOnInit(): void {
+    this.findAllPendingTickets();
+    this.authService.User.subscribe(user1 => {
+      this.user = user1;
+    });
+  }
+
+  user: any;
+  allPendingTickets: Ticket[] = [];
+  allMyTickets: Ticket[] = [];
+  tempTicket:Ticket = new Ticket(0,'0','0',new Date(),new Date(),"", "", "", "","", "", 0, "", "", "","");
+  displayPending:boolean = true;
+  pendings_nav_color = "orange";
+  accepted_nav_color = "grey";
+  errorMessage:string = "Please make sure the ticket is 'In progress' and have a edited video.";
+  hasError:boolean = false;
+  tempUploadedVideo: any;
+  
+
+  toggleDisplayPending(b:boolean){
```

Comment:
This is similar to another implementation that I saw earlier. Is this boolean value passed down from a parent component? In any case, it still makes sense to use the displayPending property since it already points to whatever value "b" points to.

Reply:
Changed 'b' to 'displayPending'.